lib/movable_ink/aws/ec2.rb

```diff
@@ -106,6 +107,64 @@ def instances(role:, exclude_roles: [], region: my_region, availability_zone: ni
         end
       end
 
+      def instances_with_consul_discovery(role:, region: my_region, availability_zone: nil)
+        if role == nil || role == ''
+          raise MovableInk::AWS::Errors::RoleNameRequiredError
+        end
+
+        if role.include? '_'
```

Comment:
Can we stick with normal method invocation style?
```suggestion
        if role.include?('_')
```

Reply:
Got it.